lib/usdUfe/ufe/UsdClipboardCommands.cpp

```diff
@@ -484,12 +488,22 @@ void UsdPasteClipboardCommand::execute()
             appendToVector(duplicateCmd->targetItems(), pasteInfo.successfulPastes);
             auto tmpTargetItems = duplicateCmd->targetItems();
             _targetItems.insert(_targetItems.end(), tmpTargetItems.begin(), tmpTargetItems.end());
+            itemsToSelect.insert(itemsToSelect.end(), tmpTargetItems.begin(), tmpTargetItems.end());
         }
 
         // Add the paste info.
         _pasteInfos.push_back(std::move(pasteInfo));
     }
 
+    if (!itemsToSelect.empty()) {
+        // Select the newly pasted items. This matches native DCC (ex: Maya) behavior.
+        Ufe::Selection pasteSelect;
+        for (const auto& item : itemsToSelect) {
+            pasteSelect.append(item);
+        }
+        Ufe::GlobalSelection::get()->replaceWith(pasteSelect);
```

Comment:
Is it okay to select this way in a command? Normally I would have used [UfeSelectionUndoItem::select()](https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/undo/OpUndoItems.cpp#L413), but that is in MayaUsd (not UsdUfe).

Reply:
Note: according to the JIRA we only select the normal prims (ignoring materials and shaders for now).